backend/src/db/migrations/20230822132525_create_articles_table.ts

```diff
@@ -0,0 +1,48 @@
+import { type Knex } from 'knex';
+
+const TableName = {
```

Comment:
Yep, we can,, and for database enum it may not be something bad (cause we won't want to change any database tables names),, but overall it's better to don't use anything from outer code in migrations

Reply:
Okay, we can left it as it is)

Reply:
Ok.